src/Microsoft.FeatureManagement/ServiceCollectionExtensions.cs

```diff
@@ -33,7 +34,17 @@ public static IFeatureManagementBuilder AddFeatureManagement(this IServiceCollec
 
             services.AddScoped<IFeatureManagerSnapshot, FeatureManagerSnapshot>();
 
-            return new FeatureManagementBuilder(services);
+            services.TryAddSingleton<ITargetingContextAccessor, EmptyTargetingContextAccessor>();
+
+            var builder = new FeatureManagementBuilder(services);
+
+            builder.AddFeatureFilter<PercentageFilter>();
+
+            builder.AddFeatureFilter<TimeWindowFilter>();
+
+            builder.AddFeatureFilter<TargetingFilter>();
```

Comment:
We discussed only adding ContextualTargetingFilter by default since TargetingFilter requires ITargetingContextAccessor implementation.

Reply:
```
services
    .AddFeatureManagement()
    .WithTargeting<MyTargetingContextAccessor>(); 
```

Reply:
Yes. This commit was before the meeting on last Thursday. I did not update it.